plugin/db/sqlite/sqlite.go

```diff
@@ -123,12 +123,90 @@ func (driver *Driver) SyncSchema(ctx context.Context) ([]*db.User, []*db.Schema,
 
 		var schema db.Schema
 		schema.Name = dbName
+
+		sqldb, err := driver.GetDbConnection(ctx, dbName)
+		if err != nil {
+			return nil, nil, fmt.Errorf("failed to get database connection for %q: %s", dbName, err)
+		}
+		txn, err := sqldb.BeginTx(ctx, &sql.TxOptions{ReadOnly: true})
+		if err != nil {
+			return nil, nil, err
+		}
+		defer txn.Rollback()
+
 		// TODO(d-bytebase): retrieve database schema such as tables and indices.
```

Comment:
Remove this comment?

Reply:
This is still needed till we have index sync in place.